lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1917,7 +1922,8 @@ def plot_trisurf(self, *args, color=None, norm=None, vmin=None, vmax=None,
         Other arguments are passed on to
```

Comment:
Same text block in parameters section problem. We work around that by summarizing this under a `**kwargs` parameter:

~~~
        **kwargs
            All other keyword argument are passed to 
            `~mpl_toolkits.mplot3d.art3d.Poly3DCollection`.
~~~